qCC/ccCommandLineCommands.cpp

```diff
@@ -2584,31 +2584,61 @@ bool CommandRemoveSF::process(ccCommandLineInterface &cmd)
 {
 	if (cmd.arguments().empty())
 	{
-		return cmd.error(QObject::tr("Missing parameter: SF index after %1").arg(COMMAND_REMOVE_SF));
+		return cmd.error(QObject::tr("Missing parameter: SF index or SF name after %1").arg(COMMAND_REMOVE_SF));
 	}
 
-	bool paramOk = false;
+	int sfIndex = -1;
+	bool validSFIndex = true;
 	QString sfIndexStr = cmd.arguments().takeFirst();
-	int sfIndex = sfIndexStr.toInt(&paramOk);
-	if (!paramOk)
+	if (sfIndexStr.toUpper() == OPTION_LAST)
 	{
-		return cmd.error(QObject::tr("Failed to read a numerical parameter: SF index. Got '%1' instead.").arg(sfIndexStr));
+		sfIndex = -2;
+		cmd.print(QObject::tr("Set active SF index: LAST"));
 	}
-	cmd.print(QObject::tr("\tSF index: %1").arg(sfIndex));
-
-	if (sfIndex < 0)
+	else
 	{
-		return cmd.error(QObject::tr("Invalid SF index (positive value expected)"));
+		sfIndex = sfIndexStr.toInt(&validSFIndex);
+		if (!validSFIndex)
+		{
+			sfIndex = -1;
+			if (sfIndexStr.size() >= 2 && sfIndexStr.startsWith('\''))
```

Comment:
maybe we should move this code in a kind of 'helper function' and reuse it elsewhere? (I believe other methods offer this possibility to deduce the SF from its name now).

Reply:
Good suggestion, I will have a closer look to it. Thanks for the feedback!

Reply:
I have added a generic function to parse the SF index / name